### PR TITLE
Setup .button to match button styles

### LINKED ISSUE #153: Combine styling of buttons and remove role="button"
There are a number of duplicate rules for styling buttons and their interaction states: the `<button>` element, form inputs, and a `.button` class with its own similar set of rules.

We should combine these where appropriate and remove styling for `role="button"` in favor of the `.button` class.

For reference, some issues with `role="button"`:
- [Needs JavaScript added to handle key events](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#accessibility_concerns)—not "simple"
- Developers might use it instead of the `.button` class to style links as buttons (see #152)

Developers can still add button styling using the `.button` class; the goal is by not providing styling for `role="button"` developers would be less likely to use the button role outside of its intended purpose, e.g. to style links like buttons.

### PR DESCRIPTION
**Pull request by pietvanzoen**
2nd commit here consolidates the `.button`/`button` styling. Let me know if you'd rather this be different.

Fixes #153

### CODE REVIEW

File: simple.css [hunk 1]
```diff
@@ -179,7 +179,8 @@ a:hover {
 }
 
 button,
-[role="button"],
+.button,
+a.button, /* extra specificity to override a */
```
Comment (lkhrs) [inline simple.css:183]: I removed `a.button` in my tests and didn't see a change, can you confirm?

Comment (pietvanzoen) [reply simple.css:183]: Yup... you're right. not sure what was going on there.

**Comment by pietvanzoen** [comment]
Thanks for the tweaks! looks good to me. :)

**Review by lkhrs — COMMENTED** [review]
Turns out we did need that extra selector after all—Safari had specificity issues but only on the Netlify preview, weird. 

Merging

**Comment by pietvanzoen** [comment]
> Turns out we did need that extra selector after all—Safari had specificity issues but only on the Netlify preview, weird. 
> 
> Merging

Huh! So I wasn't imagining things. 😛 I thought I'd done it for a reason but it seemed fine when I checked. 

Nice catch

**Pull request merged**